subprojects/gradle/runner/service/src/test/kotlin/com/avito/runner/service/worker/device/adb/AdbDeviceParserTest.kt

```diff
@@ -0,0 +1,38 @@
+package com.avito.runner.service.worker.device.adb
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.jupiter.api.Test
+
+internal class AdbDeviceParserTest {
+
+    val parser = AdbDeviceParser()
+
+    @Test
+    fun `parse correctly`() {
```

Comment:
naming...

![image](https://user-images.githubusercontent.com/1105133/74967191-90ec0280-5429-11ea-92c2-4a8f330425a3.png)

Reply:
pizdec problema

Reply:
@warabei14 , let's discuss it. We think that naming conventions in tests are worth it and help us to maintain them cheaper.